pmoves/services/clap-embed/app.py

```
@@ -0,0 +1,106 @@
+    @app.on_event("startup")
+    async def _start_nats():
+        # When NATS_URL is configured, subscribe the responder to
+        # audio.embed.request.v1 so the advertised NATS path actually works.
+        # NATS is optional — failure to connect must not break the HTTP service.
+        app.state.nats_conn = None
+        if not Config.NATS_URL:
+            return
+        try:
+            from nats_responder import run_responder
+            app.state.nats_conn = await run_responder(get_embedder())
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)